shared/src/main/java/uk/gov/di/authentication/shared/services/DynamoAuthenticationAttemptsService.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.di.authentication.shared.services;
+
+import uk.gov.di.authentication.shared.entity.AuthenticationAttempts;
+import uk.gov.di.authentication.shared.helpers.NowHelper;
+
+import java.time.Clock;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+
+public class DynamoAuthenticationAttemptsService extends BaseDynamoService<AuthenticationAttempts> {
+
+    public DynamoAuthenticationAttemptsService(ConfigurationService configurationService) {
+        super(AuthenticationAttempts.class, "authentication-attempts", configurationService);
+        new NowHelper.NowClock(Clock.systemUTC());
```

Comment:
From my eyes this seems to throw away the `NowClock` instance and could probably be removed.